packages/client/src/controllers/engine.ts

```diff
@@ -299,16 +308,41 @@ export class Engine extends IEngine {
     });
     await this.client.pairing.set(topic, pairing);
     await this.client.core.relayer.subscribe(topic);
-    // TODO(ilja) this.expirer / timeout pairing ?
+    await this.setExpiry(topic, expiry);
 
     return { topic, uri };
   }
 
   private activatePairing: EnginePrivate["activatePairing"] = async topic => {
-    await this.client.pairing.update(topic, {
-      active: true,
-      expiry: calcExpiry(THIRTY_DAYS),
-    });
+    await this.client.pairing.update(topic, { active: true });
```

Comment:
expiry was removed from here… this looks like a typo

Reply:
Its not, it is handled within `this.setExpiry`. that's called right after. Purpose of this is to have call just 1 method to simultaneously update pairing / session expiry as well as set them on expirer

Reply:
gotcha. marked as resolved